forc/GFS/README.md

```diff
@@ -0,0 +1,94 @@
+## Description
+
+* This package generates DATM input data for the forcing fluxes at the sea surface from the NCEP operational GFSv16 outputs.
+
+## Inputs of conversion in conv_gfs2datm_long_beta4.f
+
+* INPUT Sources : from the output of GFSv16,
+
+I. ./gdas.${dat}/${hh}/atmos/gdas.t${hh}z.sfluxgrbf000.grib2 :: Input data are from the GFSv16 outs of initial analysis fields, 
+  except following missing variables in the initial file.
+
+II. ./gdas.${datb}/${hhb}/atmos/gdas.t${hhb}z.sfluxgrb${hprep}.grib2 :: 6 hour forecast fields at the analysis time
+      ==> PRATE_surface, UFLX_surface, VFLX_surface
+
+III. ./gdas.${dat}/${hh}/atmos/gdas.t${hh}z.atmf000.nc  ==> delz
+
+| Input variable |	Source	| Output variable|
+| --- | --- | ---|
+| longitude	| I| lon |
+| latitude |I |lat |
+| time | I| time |
+|DLWRF_surface  |I |DLWRF |
+|ULWRF_surface  | I|ULWRF |
+| DSWRF_surface |I |DSWRF |
+|DSWRF_surface |I |vbdsf_ave |
+|DSWRF_surface |I | vddsf_ave|
+|DSWRF_surface |I |nbdsf_ave |
+|DSWRF_surface |I | nddsf_ave|
```

Comment:
looks like copy/paste error.

Reply:
Indeed all variables are converted from DAWRF. 
In Estimation and calculation : vbdsf_ave, vddsf_ave, nbdsf_ave, nddsf_ave are estimated from DSWRF_surface, multiplied by 0.285 (vbdsf_ave    , addsf_ave) and 0.215 (nbdsf_ave, nddsf_ave).

Reply:
Thanks for this information.